projects/packages/my-jetpack/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+	"extends": "jetpack-js-tools/tsconfig.base.json",
+	// List all sources and source-containing subdirs.
+	"include": [ "./_inc" ],
+	"files": [ "./global.d.ts" ]
+}
```

Comment:
This is for improved typescript compatibility. (prior to this, the types in file `global.d.ts` were not fully being loaded and recognized in VSCode)